.github/workflows/client-celo.yml

```
@@ -0,0 +1,79 @@
+name: Go / Celo
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Consider triggering the workflow on pull requests on all branches. Thanks to that the workflow would trigger also on PRs from `feature_branch_A` to `feature_branch_B`.
```suggestion
  pull_request:
```

---

Good catch! Please see: https://github.com/keep-network/keep-ecdsa/commit/03aafa389975e71405a6a3441ab777ea8c9ce46d